specs/007-elections-discovery/tasks.md

```
@@ -0,0 +1,331 @@
+
+### Within Each User Story
+
+- Types/API/hooks before components
+- Components before route integration
+- Implementation before tests (tests validate implementation)
+- Commit after each logical group
+
+### Parallel Opportunities
+
+**Phase 2 (Foundational)**:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)